public_html/pipeline.php

```diff
@@ -166,81 +153,88 @@
+        ?>
       </select>
     </div>
   </li>
   <?php endif; ?>
 </ul>
 
 <?php
-
-# echo '<pre>'.print_r($pipeline, true).'</pre>';
-# echo '<pre>'.print_r($gh_tree_json, true).'</pre>';
-
+########
+# Make a row with a column for content for everything except the stats page
+########
 if($pagetab !== 'stats'){
-    echo '<div class="row"><div class="col-lg-8 order-lg-1">';
+  echo '<div class="row"><div class="col-lg-8 order-lg-1">';
 }
+
+########
 # Print content
-if($pagetab == 'home' || $pagetab == 'output' || $pagetab == 'usage' || $pagetab == 'releases'){
-  if(preg_match('/<!-- params-docs -->/')){
-    $content = '<div class="rendered-markdown">'.preg_replace('/<!-- params-docs -->/',$schema_content,$content).'</div>';
-  } else {
-    $content = '<div class="rendered-markdown">'.$content.$schema_content.'</div>';
-  }
-  echo $content;
-}
-else {
-  echo $content;
+########
+# Add on the rendered schema docs (empty string if we don't have it)
+if(preg_match('/<!-- params-docs -->/')){
+  $content = preg_replace('/<!-- params-docs -->/', $schema_content, $content);
+} else {
+  $content .= $schema_content;
 }
+echo '<div class="rendered-markdown">'.$content.'</div>';
+
+########
+# Sidebar for everything except the stats page
+########
 if($pagetab !== 'stats'){
-    echo '</div><div class="col-lg-4 order-lg-12"><div class="side-sub-subnav sticky-top">';
+  echo '</div>'; # end of the content div
+  echo '<div class="col-lg-4 order-lg-12"><div class="side-sub-subnav sticky-top">';
+
+  # Pipeline homepage & releases - key stats
+  if(in_array($pagetab, ['', 'releases'])){
+    echo $pipeline_stats_sidebar;
+  }
+  # Documentation - ToC
+  else {
+    $toc = '<nav class="toc nav flex-column">'.generate_toc($content).'</nav>';
+    # Add on the action buttons for the parameters docs
     if($pagetab == 'usage'){
-      $toc = '<div class="btn-group mt-1" role="group">
-                    <button class="btn btn-outline-secondary collapse-groups-btn" id="toggle_details" data-toggle="collapse"  data-target=".schema-docs-help-text" aria-expanded="false"><i class="fa mr-1"></i> Toggle details</button>
-                    <button class="btn btn-outline-secondary collapse-groups-btn" id="show_hidden" data-toggle="collapse" data-target=".param_hidden" aria-expanded="false"><i class="fa mr-1"></i> Show hidden</button>
-                </div>';
-      $toc .= '<nav class="toc nav flex-column">'.generate_toc($content).'</nav>';
-        echo $toc;
-    } else if($pagetab == 'output'){
-      $toc = '<nav class="toc nav flex-column">'.generate_toc($content).'</nav>';
-        echo $toc;
-    } else {
-        echo $pipeline_stats_sidebar;
+      $toc .= '<div class="btn-group mt-1" role="group">
+                <button class="btn btn-outline-secondary collapse-groups-btn" id="toggle_details" data-toggle="collapse" data-target=".schema-docs-help-text" aria-expanded="false"><i class="fas fa-question-circle mr-1"></i> Toggle help</button>
```

Comment:
Not sure of the wording in the two toggle buttons. Either it should be "toggle" for both or "show" for both not mixed.

Reply:
Done 👍🏻